src/routeGeneration/routeGenerator.ts

```diff
@@ -48,11 +49,15 @@ export class RouteGenerator {
       if (additionalProperties) {
         // Then the model for this type explicitly allows additional properties and thus we should assign that
         return JSON.stringify(additionalProperties);
-      } else if (this.minimalSwaggerConfig.noImplicitAdditionalProperties === true) {
+      } else if (this.minimalSwaggerConfig.noImplicitAdditionalProperties === 'silently-remove-extras') {
         return JSON.stringify(false);
-      } else {
+      } else if (this.minimalSwaggerConfig.noImplicitAdditionalProperties === 'throw-on-extras') {
+        return JSON.stringify(false);
+      } else if (this.minimalSwaggerConfig.noImplicitAdditionalProperties === undefined) {
         // Since Swagger defaults to allowing additional properties, then that will be our default
         return JSON.stringify(true);
+      } else {
+        return assertNever(this.minimalSwaggerConfig.noImplicitAdditionalProperties);
```

Comment:
I'm using "exhaustiveness checking" so that if we ever add new options to `noImplicitAdditionalProperties` we can get that awareness at compile-time.